### DISCUSSION TITLE
Artifacts not opening on Chrome (with certain extensions)

### DISCUSSION THREAD

**Discussion by djjrock** [danny-avila/LibreChat#4945] [Troubleshooting]
### What happened?

I use libre chat and in chrome, i cant get artifacts to open. I turned off most extensions that cause this but still seeing some issues. When i run on Safari or others in my company use chrome, we have no issue.

### Steps to Reproduce

use anthropic w sonnet latest and artifacts on chrome

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
no log errors, see screenshot
```

### Screenshots

![Screenshot 2024-12-10 at 2 59 45 PM](https://github.com/user-attachments/assets/1a82b209-d321-4190-8072-6e0b06d8a060)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by illgitthat** [comment]
Which extensions cause this?

**Comment by djjrock** [comment]
I dont know as i haven't been able to unfix it in chrome, but would love to debug or find a fix, without testing each one by one vs all on vs all off etc.
